tensorflow/stream_executor/rocm/rocm_dnn.cc

```diff
@@ -2808,7 +2808,8 @@ static DeviceMemoryBase MaybeTransformLayout(
 }
 
 port::Status MIOpenSupport::DoConvolve(
-    dnn::ConvolutionKind kind, dnn::DataType element_type, Stream* stream,
+    dnn::ConvolutionKind kind, dnn::DataType element_type,
```

Comment:
Add '''dnn::DataType output_type''' due to the base class interface change.  It is not used in this function.  In order to verify these changes, I have built Tensorflow for ROCM successfully, though I haven't run any ROCM test.